about.html

```diff
@@ -0,0 +1,120 @@
+                    <div class="logolink">
+                      <a class="logolink1" href="#"><span class="desklogo">VR Odyssey</span></a>
+                    </div>
+                    <ul class="deskul2">
+                      <li class="desklist2"><a href="#" class="desklink1">About</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">program</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">Join</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">Sponser</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">News</a></li>
+                    </ul>
+                    <div class=desbutton><button class="deskbutton">VR Campaign</button></div>
+                </div>
+                <div class="home">
+                    <ul class="homeul">
+                      <li class="homeli"><a href="./index.html" class="desklink2"><i class="fa-solid fa-house homeicon"></a></i></li>
+                      <li class="homeli"><a href="#" class="desklink2"><i class="fa-solid fa-angle-right"></a></i></li>
+                      <li class="homeli"><a href="./about.html" class="desklink2">about</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <section class="page">
+            <p class="para">"Welcome to VR!"</p>
+            <div class="clip">
+                <h1 class="heading1">VR EVOLUTION:<br>THE NEXT CHAPTER 2023</h1>
+            </div>
+            <div class="paraborder">
+                <p class="para1">We are excited to bring together professionals from across the VR industry for this two-day event exploring the latest trends, innovations, and challenges in VR technology. From keynote speeches and panel discussions to workshops, demonstrations, and networking events, we have designed an engaging and informative program that will provide attendees with valuable insights and practical skills.</p>
+            </div>
+            <div class="venue">
+                <p class="time">Please contact us per Email for any further questions<br>about VR EVO. THE NEXT CHAPTER 2023</p>
+                <p class="place">vrodyssey2023@evo.org</p>
+            </div>
+        </section>
+    </section>
+        <section class="challenge">
+            <div class="heading2">
+               <h2 class="heading3">VR Escape Room Challenge</h2>
+               <hr class="line">
+            </div>
+            <div class="program">
+                <img class="proimg" src="./images/escape.jpg">
+                <p class="propara">A VR escape room challenge where attendees work in teams to solve puzzles and escape a virtual environment within a set time frame.</p>  
+              </div>
+        </section>
+        <hr class="lineS">
+        <section class="past">
+            <h2 class="pasthead">See the past VR Odyssey Global Confrence</h2>
```

Comment:
- [ ]  You did a great job 😃 here at adding the line below the heading like the Behance, but I see that the color of this hr is different from the one in the Behance, so please 😊 try to put the correct color be like the Behance
- [ ]  Also  I see that the width of this hr is greater than the one in the Behance, so please 😊 try to decrease the width a little to be like the Behance 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/225893846-1a772515-df30-4ece-8699-a1d0c626bb56.png)  |    ![image](https://user-images.githubusercontent.com/83360636/225893806-f68651d0-2ae9-461a-985a-774c87ef12ed.png)  |